tasks/task_14/client/sender/sender.go

```diff
@@ -0,0 +1,36 @@
+package sender
+
+import (
+	"fmt"
+	"net"
+)
+
+type Sender struct {
+	conn    net.Conn
+	scanner scanner
+}
+
+func NewSender(conn net.Conn, s scanner) *Sender {
+	return &Sender{
+		conn:    conn,
+		scanner: s,
+	}
+}
+
+type scanner interface {
+	ScanTerminal() (string, error)
```

Comment:
sender doesn't know will it be terminal or something else, but you named method with word "terminal"